File: palindrome/palindrome.py
def is_palindrome():
    word = input("input a word and we will verify if is palindrom: ").replace(" ", "")
    word_reverse = word[::-1]
    if word.casefold() == word_reverse.casefold():
        print("is palindrome")
        return
    print("is not palindrome")
    
is_palindrome()

File: palindrome/palindrome.js
const is_palindrome = () =>{
    let word = prompt("input a word and we will verify if is palindrome").replaceAll(" ", "").split("")
    let word_reverse = word.reverse()
    word = word.toString().replaceAll(",", "")
    word_reverse = word_reverse.toString().replaceAll(",", "")
    if(word === word_reverse){
        alert("is palindrome")
        return
    }
    alert("is not palindrome")
}

is_palindrome()